docker/configs/clickhouse/compression.xml

```diff
@@ -0,0 +1,8 @@
+<clickhouse>
+  <compression>
+    <!-- Use LZ4 compression for better performance -->
```

Comment:
We have a lot of columns where we override the codec to zstd, and I believe lz4 is the default.
I don't think this change is necessary.